app/api/discount_codes.py

```diff
@@ -288,3 +304,14 @@ def before_get(self, args, kwargs):
     schema = DiscountCodeSchemaEvent
     data_layer = {'session': db.session,
                   'model': DiscountCode}
+
+
+class DiscountCodeTicketRelationship(ResourceRelationship):
+    """
+    DiscountCode Ticket Relationship
+    """
+    decorators = (jwt_required,)
+    methods = ['GET', 'PATCH']
```

Comment:
Even though this class is not needed IMO but if it is needed, then remove PATCH from this. We don't allow updating an entry via a relationship. In order to update the details resource should be used.